components/root/root.jsx

```diff
@@ -206,8 +205,6 @@ export default class Root extends React.Component {
         };
         if (global.Intl) {
```

Comment:
Could you remove this check and inline `afterIntl`?

Reply:
Yeah I thought about that after pushing. I'll do it ASAP 👍

Reply:
Done